Automattic/wp-calypso
# Plans 2023 : Removed feature group innovative features and other updates

#### Proposed Changes

- Removed group innovative technologies, and reorganised features and updated changes done to the list
- Updates to group titles as shown in the design file
![image](https://user-images.githubusercontent.com/3422709/215577094-321de9e6-f93f-4003-b62b-3ecb6c2c701c.png)


#### Testing Instructions
- Go to /start/plans?flags=onboarding/2023-pricing-grid
- The groups and features should be displayed correctly in the correct order and grouping



#### Pre-merge Checklist





Fixes Automattic/martech#1433
